src/saving/NewSaveMenu.tscn

```diff
@@ -31,7 +31,7 @@ __meta__ = {
 [node name="Label" type="Label" parent="VBoxContainer"]
 margin_right = 1280.0
 margin_bottom = 23.0
-text = "Create New Save"
+text = "NEW"
```

Comment:
This needs to be more specific than `NEW`
`CREATE_NEW_SAVE` should be used to differentiate other places where `new` is used.